configurationdemo/source/c/CMakeLists.txt

```diff
@@ -33,6 +33,8 @@ add_subdirectory(libs/kaa)
 # Directory containing target support library.
 add_subdirectory(targets/${KAA_TARGET})
 
+include_directories(targets/${KAA_TARGET})
```

Comment:
If target_include_directories() executed with correct include dirs then this line is not required.